crates/aranya-daemon/example.toml

```
@@ -0,0 +1,87 @@
+# The name of the daemon used for logging and debugging
+# purposes.
+name = "name"
```

If someone sees the default value of this field output in a tracing log, defaulting to "deamon" would be more descriptive than "name".
```suggestion
name = "daemon"
```